code/modules/antagonists/blob/blobstrains/distributed_neurons.dm

```diff
@@ -29,11 +29,11 @@
 	M.apply_damage(0.6*reac_volume, TOX)
 	if(O && ishuman(M) && M.stat == UNCONSCIOUS)
 		M.death() //sleeping in a fight? bad plan.
-		var/points = rand(5, 10)
-		var/mob/living/simple_animal/hostile/blob/blobspore/BS = new/mob/living/simple_animal/hostile/blob/blobspore/weak(get_turf(M))
+	if(O && ishuman(M) && M.stat == DEAD && O.can_buy(5))
```

Comment:
seems like you'd miss a lot of "free" zombifications this way unless you keep a buffer reserve of, like, 20 blob points.

Reply:
I'm not particularly concerned.

Reply:
Instead of checking `if(O && ishuman(M)` twice, nest it on the same check